FWCore/ParameterSet/python/SequenceVisitors.py

```diff
@@ -103,6 +103,19 @@ def leave(self, visitee):
         #self._node.leave(visitee)
         self._decorated.leave(visitee)
 
+class ModuleNamesFromGlobalsVisitor(object):
```

Comment:
does this work for tasks?
We need the same capture for miniAOD modules, which is mostly all tasks.